app/src/main/java/com/fstyle/androidtrainning/UnSwipeViewPager.java

```diff
@@ -0,0 +1,46 @@
+package com.fstyle.androidtrainning;
+
+import android.content.Context;
+import android.support.v4.view.ViewPager;
+import android.util.AttributeSet;
+import android.view.MotionEvent;
+
+/**
+ * Created by huynh on 12/12/2017.
+ */
+
+public class UnSwipeViewPager extends ViewPager {
+    public UnSwipeViewPager(Context context) {
+        super(context);
+    }
+
+    public UnSwipeViewPager(Context context, AttributeSet attrs) {
+        super(context, attrs);
+    }
+
+    @Override
+    public boolean onInterceptTouchEvent(MotionEvent arg0) {
+        return false;
+    }
+
+    @Override
+    public boolean canScrollHorizontally(int direction) {
+        return false;
+    }
+
+    @Override
+    public boolean onTouchEvent(MotionEvent ev) {
```

Comment:
WARNING: [ClickableViewAccessibility] `UnSwipeViewPager#onTouchEvent` should call `UnSwipeViewPager#performClick` when a click is detected
 If a `View` that overrides `onTouchEvent` or uses an `OnTouchListener` does not also implement `performClick` and call it when clicks are detected, the `View` may not handle accessibility actions properly. Logic handling the click actions should ideally be placed in `View#performClick` as some accessibility services invoke `performClick` when a click action should occur.